crypto/aria/aria.c

```diff
@@ -1,454 +1,1289 @@
+        (T1) ^= (T2);               \
+        (T2) ^= (T3);               \
+        (T0) ^= (T1);               \
+                                    \
+        (T3) ^= (T1);               \
+        (T2) ^= (T0);               \
+        (T1) ^= (T2);               \
+    } while(0)
+
+/* Byte-level diffusion */
+#define ARIA_DIFF_BYTE(T0, T1, T2, T3)                                  \
+    do {                                                                \
+        (T1) = (((T1) << 8) & 0xff00ff00) ^ (((T1) >> 8) & 0x00ff00ff); \
+        (T2) = rotr32(T2, 16);                                          \
+        (T3) = bswap32(T3);                                             \
+    } while(0)
+
+/* Odd round Substitution & Diffusion */
+#define ARIA_SUBST_DIFF_ODD(T0, T1, T2, T3)             \
+    do {                                                \
+        ARIA_SBOX_LAYER1_WITH_PRE_DIFF(T0, T1, T2, T3); \
+        ARIA_DIFF_WORD(T0, T1, T2, T3);                 \
+        ARIA_DIFF_BYTE(T0, T1, T2, T3);                 \
+        ARIA_DIFF_WORD(T0, T1, T2, T3);                 \
+    } while(0)
+
+/* Even round Substitution & Diffusion */
+#define ARIA_SUBST_DIFF_EVEN(T0, T1, T2, T3)            \
+    do {                                                \
+        ARIA_SBOX_LAYER2_WITH_PRE_DIFF(T0, T1, T2, T3); \
+        ARIA_DIFF_WORD(T0, T1, T2, T3);                 \
+        ARIA_DIFF_BYTE(T2, T3, T0, T1);                 \
+        ARIA_DIFF_WORD(T0, T1, T2, T3);                 \
+    } while(0)
+
+/* Q, R Macro expanded ARIA GSRK */
+#define _ARIA_GSRK(RK, X, Y, Q, R)                  \
+    do {                                            \
+        SET_U32(RK, 0,                              \
+            ((X)[0]) ^                              \
+            (((Y)[((Q)    ) % 4]) >> R) ^           \
+            (((Y)[((Q) + 3) % 4]) << (32 - (R))));  \
+        SET_U32(RK, 1,                              \
+            ((X)[1]) ^                              \
+            (((Y)[((Q) + 1) % 4]) >> R) ^           \
+            (((Y)[((Q)    ) % 4]) << (32 - (R))));  \
+        SET_U32(RK, 2,                              \
+            ((X)[2]) ^                              \
+            (((Y)[((Q) + 2) % 4]) >> R) ^           \
+            (((Y)[((Q) + 1) % 4]) << (32 - (R))));  \
+        SET_U32(RK, 3,                              \
+            ((X)[3]) ^                              \
+            (((Y)[((Q) + 3) % 4]) >> R) ^           \
+            (((Y)[((Q) + 2) % 4]) << (32 - (R))));  \
+    } while(0)
+
+#define ARIA_GSRK(RK, X, Y, N) _ARIA_GSRK(RK, X, Y, 4 - ((N) / 32), (N) % 32)
+
+#define ARIA_DEC_DIFF_BYTE(X, Y, TMP)           \
+    do {                                        \
+        (TMP) = rotr32((X), 8);                 \
+        (Y) = (TMP) ^ rotr32((X) ^ (TMP), 16);  \
+    } while(0)
+
+void aria_encrypt(const unsigned char *in, unsigned char *out,
+    const ARIA_KEY *key) 

```

Comment:
In such cases we align continuation with (. For example
```
void aria_encrypt(const unsigned char *in, unsigned char *out,
                  const ARIA_KEY *key) 
```
[I'm not sure how it will look in notification e-mail, do see this page.]